src/jit-format/jit-format.cs

```diff
@@ -563,6 +568,54 @@ public static void ParseBuildLog(string buildLog, string compileCommands, string
             }
         }
 
+        public static string rewriteCompileCommands (string compileCommandFile, string project)
+        {
+            string[] compileCommands = File.ReadAllLines(compileCommandFile);
```

Comment:
I can see that only relatively primitive parsing is required at this point, but why not parse as JSON?